### ISSUE TITLE
[Bug]: logTab is not a constructor when accessing Datahub Config

### ISSUE THREAD

**Issue by sessterling** [pimcore/data-importer#344]
### Expected behavior

When a "Data Object Importer" configuration is created, an interface is displayed to configure it.

### Actual behavior

When creating a "Data Object Importer" configuration, the screen remains blank.

I receive the error "Uncaught TypeError: pimcore.plugin.pimcoreDataImporterBundle.configuration.components.logTab is not a constructor" when trying to access the Datahub Config window.

![Capture d’écran du 2023-07-10 12-02-21](https://github.com/pimcore/data-hub/assets/28145524/f057b994-23c9-4caa-9d0e-24fdf4afa298)

The Datahub Config window is then stuck and is not interactive

![Capture d’écran du 2023-07-10 12-04-46](https://github.com/pimcore/data-hub/assets/28145524/93a23367-89ff-46e8-93c7-e36aa5646551)

### Steps to reproduce

Install Pimcore Version: v11.0.3

Install Datahub

Install Data Importer

**Comment by fashxp** [comment]
Eventually application logger is not installed/activated. 
Should be done automatically though during install

**Comment by sessterling** [comment]
I enable the PimcoreApplicationLoggerBundle in my bundle.php 

`<?php

use Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle;

return [
    PimcoreApplicationLoggerBundle::class => ['all' => true]
]
`
the package was already installed

![Capture d’écran du 2023-07-10 14-31-04](https://github.com/pimcore/data-hub/assets/28145524/56f592c3-4614-4096-b39b-d07315679502)

**Comment by robertSt7** [comment]
@sessterling Hi, sorry I am only able to reproduce the bug when I uninstall `PimcoreApplicationLoggerBundle`. Could you please try to re-install the bundle? Thanks

**Comment by sessterling** [comment]
@robertSt7 Hi, i unistall `PimcoreApplicationLoggerBundle` (bin/console pimcore:bundle:uninstall PimcoreApplicationLoggerBundle) and install again (bin/console pimcore:bundle:install PimcoreApplicationLoggerBundle) but the error is still present

I also have this error when i login 

![Capture d’écran du 2023-07-17 09-15-15](https://github.com/pimcore/data-hub/assets/28145524/dbddb5d1-1f36-439f-bbbd-046dd7525874)

**Comment by dvesh3** [comment]
@robertSt7 please check if the dependent (`ApplicationLogger`) bundle is properly installed and registered here.

**Comment by robertSt7** [comment]
@sessterling Hi, sorry, but we are closing the issue as we are not able to reproduce it. Please feel free to open it again if you have a reproducible case. Thanks for your understanding

**Comment by sessterling** [comment]
For people who have the same problem, the solution is to first import PimcoreApplicationLoggerBundle, then PimcoreDataHubBundle in config\bundles.php. 

`<?php

use Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle;
use Pimcore\Bundle\DataImporterBundle\PimcoreDataImporterBundle;
use Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle;

return [
    Pimcore\Bundle\SimpleBackendSearchBundle\PimcoreSimpleBackendSearchBundle::class => ['all' => true],
    Pimcore\Bundle\TinymceBundle\PimcoreTinymceBundle::class => ['all' => true],
    PimcoreApplicationLoggerBundle::class => ['all' => true],
    PimcoreDataHubBundle::class => ['all' => true],
    PimcoreDataImporterBundle::class => ['all' => true]
];
`

**Comment by wujku** [comment]
The problem also happened to me.

I did not register the `PimcoreApplicationLoggerBundle` package in the `config/bundles.php` file, because it is loaded in `PimcoreDataImporterBundle::registerDependentBundles` by default.

```php
<?php

use Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle;
use Pimcore\Bundle\DataImporterBundle\PimcoreDataImporterBundle;

return [
    PimcoreDataHubBundle::class => ['all' => true],
    PimcoreDataImporterBundle::class => ['all' => true],
];
```

Unfortunately, because loading priority has not been defined for `PimcoreApplicationLoggerBundle` as second parameter of `addBundle` method, `PimcoreApplicationLoggerBundle` is loaded after `PimcoreDataImporterBundle`. 

Error `Uncaught TypeError: Cannot read properties of undefined (reading 'applicationlogger')` is a consequence of js libraries concatenation order for Pimcore script-proxy file.

https://github.com/pimcore/data-importer/blob/1.8/src/PimcoreDataImporterBundle.php#L154

```php
    public static function registerDependentBundles(BundleCollection $collection): void
    {
        $collection->addBundle(PimcoreDataHubBundle::class, 20);
        $collection->addBundle(new FlysystemBundle());
        $collection->addBundle(new PimcoreAdminBundle(), 60);

        if (\Pimcore\Version::getMajorVersion() >= 11) {
            // Missed priority
            $collection->addBundle(
                \Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle::class
            );
        }
    }
```

When I defined priority to `1`, the logger's js code has been initialized earlier, so the functionality works correctly. @dvesh3 in my opinion this should be corrected in the bundle code as below:

```php
$collection->addBundle(\Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle::class, 1);
```

Temporary solution:

`config/bundles.php`

```php
<?php

use Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle;
use Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle;
use Pimcore\Bundle\DataImporterBundle\PimcoreDataImporterBundle;

return [
    PimcoreApplicationLoggerBundle::class => ['all' => true],
    PimcoreDataHubBundle::class => ['all' => true],
    PimcoreDataImporterBundle::class => ['all' => true],
];
```

**Comment by fashxp** [comment]
we need to set priority of application logger bundle properly.

**Comment by dvesh3** [comment]
Fixed by #351
